shelley/design-spec/delegation_design_spec.tex

```diff
@@ -1168,44 +1096,44 @@ \subsubsection{Stake Pool Registration Certificates}
 
 \begin{itemize}
 \item
-  The public key of the pool, \(vks_\text{pool}\).
+  The public key of the pool, \(vk_\text{pool}\).
 
 \item
-  A hash \(\mathcal{H}(vks_\text{reward})\) of a staking key. Usually, this will
-  be a \emph{registered} staking key, controlled by the pool operator. The
-  rewards for the pool operator will be paid to the reward account associated
-  with this \emph{reward staking key} \(vks_\text{reward}\)\footnote{We do
-    \emph{not} want rewards to be paid to the key of the pool itself. An
-    important reason for this is that for security reasons, stake pool
-    operators are required to use operational keys as described in
-    \cref{operational-key-certificates}, while storing the key of the pool
-    securely offline. Requiring the pool key for withdrawing rewards would be
-    detrimental to this.}.
+  A stake address \(\mathcal{A}_{s,\text{reward}}\), called the \emph{reward
+    address} for the pool. Usually, this will be a \emph{registered} stake
+  address, controlled by the pool operator. The rewards for the pool operator
+  will be paid to the reward account of
+  \(\mathcal{A}_{s,\text{reward}}\)\footnote{We do \emph{not} want to associate
+    the reward account with the key of the pool itself. An important reason for
+    this is that for security reasons, stake pool operators are required to use
+    operational keys as described in \cref{operational-key-certificates}, while
+    storing the key of the pool securely offline. Requiring the pool key for
+    withdrawing rewards would be detrimental to this.}.
 
   If a pool operator wants to donate their rewards to a charity, they can do so
-  by supplying the hash of a staking key that is controlled by that charity as
-  the reward staking key. They can then advertise to other stakeholders that
-  they are doing so (although evidence that the staking key does indeed belong
-  to the charity has to be provided out of band).
+  by using a stake address that is controlled by that charity as the reward
+  address. They can then advertise to other stakeholders that they are doing so
+  (although evidence that the stake address does indeed belong to the charity
+  has to be provided out of band).
```

Comment:
Does this matter from the perspective of delegation design?  Should it be a description of a general mechanism rather than specific to charities?